src/extension/ui/src/types/Catalog.ts

```diff
@@ -0,0 +1,44 @@
+export interface MCPServer {
+    Prompts: number;
+    Resources: Record<string, any> | null;
+    Tools: Tool[] | null;
+    Config: Config[] | null;
+    Secrets: Secret[] | null;
+    Examples: any[] | null;
+}
+
+export interface Tool {
+    Name: string;
+    Description: string;
+}
+
+export interface Config {
```

Comment:
This type does not sufficient for what Config actually is.  It's a named json schema Object that comes from MCP authors. This type doesn't represent the underlying thing we're modeling.

Reply:
These two files were not meant to be committed, thanks for catching.